test/frontend/MembershipPlan.test.js

```diff
@@ -8,42 +8,86 @@ import { membershipPlans } from '../../src/Enums'
 
 import MembershipPlan from '../../src/Pages/MembershipApplication/MembershipPlan'
 import Adapter from 'enzyme-adapter-react-16'
+import { mockMonth, revertClock } from '../mocks/Date'
 
 Enzyme.configure({ adapter: new Adapter() })
 
 describe('<MembershipPlan />', () => {
-  const SEMESTER_MEMBERSHIP = 'Spring 2020'
-  const YEAR_MEMBERSHIP = 'Spring and Fall 2020'
+  after(done => {
+    // get rid of the stub
+    revertClock()
+    done()
+  })
+
+  const year = new Date().getFullYear()
+  const SEMESTER_MEMBERSHIP_FALL = 'Fall ' + year
```

Comment:
can you change this to `FALL_SEMESTER_MEMBERSHIP`? otherwise the developer has to read the entire variable to determine which type it is. do the same for the other constants.